src/rgw/rgw_sal_motr.cc

```diff
@@ -4608,21 +4605,20 @@ int MotrMultipartWriter::complete(size_t accounted_size, const std::string& etag
   encode(attrs, bl);
   part_obj->meta.encode(bl);
 
-  string p = "part.";
-  char buf[32];
-  snprintf(buf, sizeof(buf), "%08d", (int)part_num);
-  p.append(buf);
-  string tenant_bkt_name = get_bucket_name(head_obj->get_bucket()->get_tenant(), head_obj->get_bucket()->get_name());
```

Comment:
`tenant_bkt_name` is required by `update_bucket_stats()` called later in the function.

Reply:
Fixed. Thanks!